tekton/cronjobs/dogfooding/images/skopeo-nightly/cronjob.yaml

```diff
@@ -22,3 +22,5 @@ spec:
               value: gcr.io/tekton-releases/dogfooding/skopeo:latest
             - name: CONTEXT_PATH
               value: tekton/images/skopeo
+            - name: PLATFORMS
+              value: "linux/amd64,linux/s390x,linux/ppc64le,linux/arm64"
```

Comment:
I see platform `linux/arm64` in few yamls any reason for that ?

Reply:
At this moment pipeline nightly builds are done for amd64, s390x, ppc64le and arm64. So I thought it would be nice to have also arm64 dogfooding images where it's possible. For some images it's not possible, because proposed multi-arch build requires corresponding availability of base images for same architectures, and for arm64 some base images are not available